docs/sources/setup/install/helm/install-microservices/_index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Install the microservice Helm chart 
+menuTitle: Install microservice Loki
+description: Installing Loki in microservice (distributed) mode using the Helm chart.
+aliases:
+  - ../../../installation/helm/microservice/
+  - ../../../installation/helm/install-microservice/
+weight: 300
+keywords: 
+---
+
+# Install the microservice Helm chart
+<!-- vale Grafana.Quotes = NO -->
+<!-- vale Grafana.Quotes = YES -->
+
+This Helm Chart deploys Grafana Loki on Kubernetes.
+
+This chart configures Loki to run Loki in [microservice / distributed mode]({{< relref "../../../../get-started/deployment-modes#microservices-mode" >}}), highly available architecture designed to work with AWS S3 object storage. 
+
+The default Helm chart deploys the following components:
```

Comment:
Whenever you have a list of items, unless there's a really good reason not to, you should alphabetize the list.